docker/api/network.py

```diff
@@ -48,7 +50,8 @@ def inspect_network(self, net_id):
     @check_resource
     @minimum_version('1.21')
     def connect_container_to_network(self, container, net_id,
-                                     aliases=None, links=None):
+                                     ip4=None, ip6=None, aliases=None,
```

Comment:
Let's call those `ipv4_address` an `ipv6_address`. The additional characters are worth the clarity they provide, I think.